extensions/wikia/PortableInfobox/controllers/PortableInfoboxParserTagController.class.php

```diff
@@ -172,18 +172,25 @@ private function handleXmlParseError( $errors, $xmlMarkup ) {
 		return [ $renderedValue, 'markerType' => 'nowiki' ];
 	}
 
-	private function getThemeWithDefault( $params, PPFrame $frame ) {
-		$value = isset( $params[ 'theme-source' ] ) ? $frame->getArgument( $params[ 'theme-source' ] ) : false;
-		$themeName = $this->getThemeName( $params, $value );
+	private function getThemes( $params, PPFrame $frame ) {
+		$themes = [ ];
 
-		//make sure no whitespaces, prevents side effects
-		return Sanitizer::escapeClass( self::INFOBOX_THEME_PREFIX . preg_replace( '|\s+|s', '-', $themeName ) );
-	}
+		if ( isset( $params[ 'theme' ] ) && !empty( trim( $params[ 'theme' ] ) ) ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "theme" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)